### ISSUE TITLE
PDF thumbnails cannot be right-clicked to open a context menu when Style Settings is enabled

### ISSUE THREAD

**Issue by RyotaUshio** [kepano/obsidian-minimal#702]
### Is the bug present when using the default Obsidian theme?

No

### Is the bug present when snippets and plugins are disabled?

No

### Minimal theme version

7.5.1

### Describe the bug

**When both Minimal & Style Settings are enabled** (this is why I answered "No" to the second question), PDF thumbnails cannot be right-clicked to open a context menu.

Screen captures from the sandbox vault:

- When Minimal & Style Settings are enabled with all the options kept default

  https://github.com/kepano/obsidian-minimal/assets/72342591/c953e274-8ac8-4867-a1a5-c124938a7d38

- When Minimal is enabled but Style Settings is disabled

  https://github.com/kepano/obsidian-minimal/assets/72342591/0e65ff9d-a506-4a1c-b1fd-f87d0013bd25

- When the default theme is used and Style Settings is enabled

  https://github.com/kepano/obsidian-minimal/assets/72342591/cefea785-fce4-4b48-be32-d6423a6b01f0

### Debug info

```
SYSTEM INFO:
	Obsidian version: v1.5.8
	Installer version: v1.5.3
	Operating system: Darwin Kernel Version 22.6.0: Wed Jul  5 22:22:52 PDT 2023; root:xnu-8796.141.3~6/RELEASE_ARM64_T8103 22.6.0
	Login status: logged in
	Catalyst license: none
	Insider build toggle: off
	Live preview: on
	Base theme: adapt to system
	Community theme: Minimal v7.5.1
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 1
	Plugins enabled: 1
		1: Style Settings v1.0.7

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Comment by RyotaUshio** [comment]
I noticed that this is not specific to PDF thumbnails but applies to all images.
But for images embedded in markdown files, it is not a big deal because Obsidian doesn't show context menus by default.
However, it IS problematic when it comes to PDF thumbnails because right-clicking should trigger a context menu (copy link to page).

**Comment by kepano** [comment]
Fixed in 7.5.2

**Comment by RyotaUshio** [comment]
Thank you for the quick work!
